src/axom/sidre/docs/sphinx/parallel_io_concepts.rst

```diff
@@ -110,48 +109,51 @@ The core functionality of IOManager is contained in the ``write()`` and ``read()
-The first argument is a pointer to a group that contains no child groups or
-views, and the information in the root file is used to identify the files that
-each processor will read to load data into the argument group.
 
-The ``write()`` and ``read()`` methods above are sufficient to do a restart save/load
-when the data is the group is completely owned by the Sidre data structures.
-If Sidre is used to manage data that is externally allocated, the loading
-procedure requires some additional steps to restore data in the same
+The ``read()`` methods is called in parallel with the root file as an argument.
+It must be called on a run with the same processor count as the run that 
+called the ``write()`` method. The first argument is a pointer to a group that 
+contains no child groups and owns no views. The second argument is the name of
+the root file; the information in the root file is used to identify the files 
+that each processor will read to load data into the argument group.
+
+The ``write()`` and ``read()`` methods above are sufficient to do a restart 
+save/load when the data in the group hierarchy is completely owned by the 
+Sidre data structures. If Sidre is used to manage data that is externally 
+allocated (i.e., the hiearchy contains *external* views), the loading
+procedure requires additional steps to restore data in the same
 externally-allocated state.
 
 First the ``read()`` method is called, and the full hierarchy structure of the
-group is loaded into the Sidre Group, but no data is allocated for Views
-identified as external.  Then the calling code can examine the group and
-allocate data for the external Views.  ``View::setExternalDataPtr()`` is used
-to associate the pointer with the view.  Once this is done, IOManager's
-``loadExternalData()`` can be used to load the data from the file into the
-user-allocated arrays.
-
-Below is a code example for loading external data.  We assume that this code
+group is loaded, but no data is allocated for views identified as external. 
+Then, the calling code can examine the group and allocate data for the external
+views, as needed.  The ``View`` class method ``setExternalDataPtr()`` is used
+to associate the pointer with the view.  Once this is done, the I/O manager
+method ``loadExternalData()`` can be used to load the data from the file into 
+the user-allocated arrays.
+
+Below is a code example for loading external data. We assume that this code
 somehow has knowledge that the root group contains a single external view at
-the location "fields/external_array" describing an array of doubles.  See the
-Group and View documentation for information about how to query the Sidre data
-structures for this type of information when the code does not have a priori
-knowledge.
+the location "fields/external_array" describing an array of doubles. See the
+`Sidre API Documentation <../../../../doxygen/html/sidretop.html>`_ for details
```

Comment:
> See the `Sidre API Documentation <../../../../doxygen/html/sidretop.html>`_ for details

(I commented on a previous link to the doxygen that a link was not properly resolved).

This one *does* link to the expected doxygen page: 
https://axom.readthedocs.io/en/rhornung67-doc-updates/axom/sidre/docs/sphinx/parallel_io_concepts.html#iomanager-class-use

There's probably a subtle difference.

Reply:
see my comment on the broken link fix.